Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/PresenceTracingRiskCalculator.kt

```diff
@@ -0,0 +1,52 @@
+package de.rki.coronawarnapp.presencetracing.risk
+
+import javax.inject.Inject
+
+class PresenceTracingRiskCalculator @Inject constructor(
```

Comment:
Could be `@Reusable`?

Reply:
That would also mean the same config is used. Do you think we should use the same config?

Reply:
> That would also mean the same config is used. 

Why :thinking: ?

> Do you think we should use the same config?

We should use the same config "per operation", i.e. if we are currently doing a risk calculation, we should use the same config for all parts of the operation. Once the calculation is done, the next calculation may use a new config, but again it should not have the chance to change during calculation within the same operation.

Reply:
The config is requested once per instance, so the whole calculation uses the same config.

Reply:
You are right, it would use same one, and upwards towards `CheckInWarningMatcher` no extra config is used.